code/search/src/README.md

```diff
@@ -1,2 +1,2 @@
-# cosmos
-Your personal library of every algorithm and data structure code that you will ever encounter
+# Searching algorithms
```

Comment:
There needs to be more data added for searching in this file. Currently, it is really less. Also, take what you removed and add it to the bottom of the file.